README.md

```diff
@@ -120,9 +120,35 @@ spdy.createServer(options, function(req, res) {
   stream.end('alert("hello from push stream!");');
 
   res.end('<script src="/main.js"></script>');
-}).listen(443);
+}).listen(3000);
 ```
 
+The method is also avaliable when using SPDY with an Express server:
+
+```javascript
+var app = express();
+
+var server = spdy.createServer(options, app);
+
+app.get('*', function(req, res) {
```

Comment:
Now that I think about it, I agree with you. We most likely want to serve "main.js" to the homepage, not to every single page